src/keys.cc

```diff
@@ -167,9 +200,13 @@ String key_to_str(Key key)
 
     switch (key.modifiers)
     {
-    case Key::Modifiers::Control:    res = "c-" + res; named = true; break;
-    case Key::Modifiers::Alt:        res = "a-" + res; named = true; break;
-    case Key::Modifiers::ControlAlt: res = "c-a-" + res; named = true; break;
+    case Key::Modifiers::Control:         res = "c-" + res; named = true; break;
```

Comment:
now that we have all those combination, a loop on Control, Alt, and Shift that prepend respectively "c-", "a-", and "s-" would be cleaner.